src/install-elixir.ps1

```diff
@@ -18,4 +18,7 @@ $ProgressPreference="Continue"
 Write-Output "Installed Elixir version follows"
 & "${DIR_FOR_BIN}/bin/elixir" "-v" | Write-Output
 
+$ProgressPreference="Continue"
+New-Item "%UserProfile%\.mix\escripts" -ItemType Directory | Out-Null
```

Comment:
```suggestion
New-Item "%UserProfile%/.mix/escripts" -ItemType Directory | Out-Null
```

Reply:
This works and reduces the number of differences to the Linux script we use to have similar behaviour.